xilem_web/web_examples/elm/src/main.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use xilem_core::map_action;
+use xilem_web::{
+    document_body,
+    elements::html as el,
+    interfaces::{Element, HtmlDivElement},
+    Action, App,
+};
+
+// ----- ------
+//    Model
+// ----- ------
+
+#[derive(Debug, Default)]
+struct Model {
+    count: i32,
+}
+
+// ----- ------
+//   Message
+// ----- ------
+
+#[derive(Debug)]
+enum Message {
+    Increment,
+    Decrement,
+}
+
+impl Action for Message {}
+
+// ----- ------
+//    Update
+// ----- ------
+
+fn update(model: &mut Model, message: Message) {
+    log::debug!("Update model {model:?} by {message:?}");
+    match message {
+        Message::Increment => model.count += 1,
+        Message::Decrement => model.count -= 1,
+    }
+    log::debug!("Model updated: {model:?}");
+}
+
+// ----- ------
+//    View
+// ----- ------
+
+fn view(model: &mut Model) -> impl HtmlDivElement<Model> {
```

Comment:
I think I get why this function is called `view`, though just to be consistent with the other examples (and those in xilem itself), also because technically the updating logic is contained within this function.
```suggestion
fn app_logic(model: &mut Model) -> impl HtmlDivElement<Model> {
```